requirements.txt

```diff
@@ -3,3 +3,4 @@ web3 == 4.5.0
 requests == 2.18.4
 python-dateutil == 2.6.1
 websockets == 6.0.0
+python-kucoin == 0.1.12
```

Comment:
I don't really like the fact to add a specific exchange dependency here

Reply:
Is there a way to make it optional ?

Reply:
per our discussion in chat, do you think that's OK to go with this approach for the moment? thanks!